test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -1440,17 +1439,25 @@ public void handleUnBootstrappedSequencer() throws Exception {
      * The cluster however is healthy. Status query:
      * PORT_0 and PORT_1 are DOWN. Cluster status: STABLE
      *
+     * A few entries are appended on a Stream. This data is written to PORT_1 and PORT_2.
+     *
      * STEP 3: Client connections are restored from the previous step. PORT_0 is failed.
      * This causes sequencer failover. But since PORT_0 was not a log unit server, the cluster is
      * still reachable. Status query:
      * PORT_0 is DOWN. Cluster status: DEGRADED.
      *
      * STEP 4: PORT_1 is failed. The cluster is non-operational now. Status query:
      * PORT_0 and PORT_1 DOWN. Cluster status: UNAVAILABLE.
+     *
+     * STEP 5: All rules are cleared so that PORT_1 is revived. Similarly PORT_0 is also revived.
+     * PORT_0 now tries to hela and add itself to the log unit chain. However, bulk reads from
+     * PORT_2 are dropped causing stateTransfer to fail leaving the segments divided. Now on a
+     * cluster status query, we see that PORT_0 in a DB_SYNCING state.
+     * PORT_0 is DB_SYNCING, PORT_1 and PORT_2 are UP. Cluster status: STABLE.
```

Comment:
If one of the nodes is in DB_syncing state should the cluster status really be STABLE, if redundancy is reduced at least for a segment.